RBDM/Record.cpp

```diff
@@ -0,0 +1,83 @@
+#include"Record.h"
+
+Record::Record()
+{
+	id = ++ID;
+}
+
+void Record::Add(char type, void* value)
+{
+	switch (type)
+	{
+	case 'i':
+	{
+		Integer *cell = (Integer*)value;
+		record.push_back(cell);
+		break;
+	}
+	case 'd':
+	{
+		Double *cell = (Double*)value;
+		record.push_back(cell);
+		break;
+	}
+	case 's':
+	{
+		String *cell = (String*)value;
+		record.push_back(cell);
+		break;
+	}
+	default:
+		break;
+	}
+}
+
+void Record::Show()
+{
```

Comment:
I suggest using virtual functions to make compiler determine which class is being proceeded now.